### PR TITLE
Reduced warnings down to two.

### PR DESCRIPTION
**Pull request by samgreen**
I wasn't able to fix them all. There's one left in AFNetworking, and another referencing `currentPopoverController`, then this project will be warning free :+1:

### CODE REVIEW

**Comment by ghost** [comment]
@samgreen 

What warnings were you seeing? The two unresolved ones you mention are the only ones I'm seeing in the project, so it's not obvious what's fixed in this PR.

screen shot 2014-06-04 at 09 40 11

**Comment by 2020Deception** [comment]
If you validate the project settings with the snapshot option, 33 compiler warnings and 1 Core Foundation warning will appear
screen shot 2014-08-30 at 5 04 48 pm

File: DuckDuckGo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1684,7 +1683,7 @@
 		83F3224F149290EF00A61BC5 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastUpgradeCheck = 0460;
+				LastUpgradeCheck = 0510;
```
Comment (chatbiggulp) [inline DuckDuckGo.xcodeproj/project.pbxproj:1686]:
```
```

``````
```[]()
``````

```
```

```
```

**Comment by subsymbolic** [comment]
Thanks for submitting this PR Sam, sorry to see it languish. We've moved active development onto another project for now, so I'm closing this off.

**Pull request closed without merging**